scripts/move_linearly.py

```diff
@@ -29,6 +31,7 @@
     time.sleep(args.wait)
 
     start_time = time.time()
+    hardware.both_axes.current_location = [0, 0]
```

Comment:
So I guess really it's a bit of a hack, but:
  - It's useful to have a script that can perform relative moves
  - The move linearly method is 'lower level' than the `draw_whatever.py` scripts so I repurposed it since it was the odd one out.
  - The script's behaviour *is* documented in the `argparse` description.

I you want, I can rename the script to `move_relative.py`?

[I don't think we will really run a 'home then move linearly' script that often. If we do want to test that method adhoc, or any other which doesn't have a script, we can always do it using the python interactive console. It feels like a better solution than trying to maintain a mammoth number of scripts. What do you think?]